pongo.go

```diff
@@ -25,6 +26,8 @@ func (templar *PongoTemplar) generateTemplate() (string, error) {
 		context[key] = value
 	}
 
+	Debug("Using context %v", context)
```

Comment:
I was not a fan the co called "debug" feature, and I am still not. There might be a sensitive information you will be printing...